services/src/test/resources/mods/no-update-mods.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bulkMetadata>
+<object pid="uuid:bf0091e0-203e-46f9-a8ad-8b0dc82f33be" >
+<mods xmlns="http://www.loc.gov/mods/v3">
```

Comment:
It will have the same effect, but could you change it to:
```
<?xml version="1.0" encoding="UTF-8"?>
<bulkMetadata>
<object pid="uuid:bf0091e0-203e-46f9-a8ad-8b0dc82f33be" >
</object>
</bulkMetadata>
```
Promise I'm not just being pedantic :) That's the format that comes out of the export tool when there is no MODS for an object, so we want to ensure that when reimporting the file it doesn't change those records.